### DISCUSSION TITLE
Reference to variable values

### DISCUSSION THREAD

**Discussion by SC-66** [grafana/grafana#65051] [Dashboards, Panels & Visualizations]
I have a variable and I try to get the value for the first selected variable value and use it in the Panel title.
Like this, Title: Event Log ${Machine.[0]} 
My variable Machine is a custom variable, with multiple select and with about 50 key/value pairs and I just want to get the first selced item and be able to write it in the panel title. But it seems to be impossible to reference variable items in the array by using some sort of index? I found an old post where it was recommended to write ${variablename.1} but that doesn't work.

![image](https://user-images.githubusercontent.com/92076711/226361358-830feca6-bd0b-49df-b282-d11568627440.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82753. Feel free to continue the discussion around this there. Thank you!
